src/main/java/com/karankumar/bookproject/backend/entity/Book.java

```diff
@@ -55,14 +60,25 @@
     private LocalDate dateStartedReading;
     private LocalDate dateFinishedReading;
 
-    @ManyToOne(cascade = {CascadeType.DETACH, CascadeType.MERGE, CascadeType.REFRESH, CascadeType.REMOVE})
+    @ManyToOne(cascade = { CascadeType.DETACH, CascadeType.MERGE, CascadeType.REFRESH, CascadeType.REMOVE })
```

Comment:
Could you undo this change? In other words, there should be no spaces before and after the curly brace